src/gretel_trainer/relational/report/report_template.html

```diff
@@ -111,21 +111,23 @@ <h2>Synthetic Data Quality Results</h2>
             <th scope="col">Cross-table SQS</th>
           </tr>
           {% for table, evaluation in presenter.evaluations.items() %}
+          {% set individual_grade = presenter.sqs_score_to_grade(evaluation.individual_sqs) %}
+          {% set cross_table_grade = presenter.sqs_score_to_grade(evaluation.cross_table_sqs) %}
           <tr>
             <td><b>{{ table }}</b></td>
             <td>
               <span class="sqs-table-score">{{ evaluation.individual_sqs }}</span>
-              <span class="label label__{{ presenter.sqs_score_to_grade(evaluation.individual_sqs).lower().replace(' ', '-') }}">
-                {{ presenter.sqs_score_to_grade(evaluation.individual_sqs) }}
+              <span class="label label__{{ presenter.css_label_format(individual_grade) }}">
```

Comment:
I only saw your ask around this after I had merged in that other PR with the js, thanks for doing this.  The `set` above is nice.